src/styles/base.scss

```diff
@@ -208,6 +229,36 @@ body {
   }
 }
 
+.btn-waves-effect {
```

Comment:
Any reason why you had to move these up in this file?

Reply:
@galipremsagar aah... actually previously it was in disorganised way so I moved the btn styles in one place.